fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -111,6 +113,8 @@
 
     private static final Logger LOGGER = getLogger(FedoraResourceImpl.class);
 
+    private static final PropertyConverter propertyConverter = new PropertyConverter();
```

Comment:
Just a note to all: there is no state in a `PropertyConverter`. We could probably make it a singleton and cut out some short-lived objects.